src/components/application_manager/test/state_controller/state_controller_test.cc

```diff
@@ -1454,9 +1451,7 @@ TEST_F(StateControllerImplTest, MoveSimpleAppToValidStates) {
     EXPECT_CALL(message_helper_mock_,
                 SendHMIStatusNotification(simple_app_, kDefaultWindowId, _));
     EXPECT_CALL(app_manager_mock_,
-                OnHMILevelChanged(simple_app_id_,
-                                  initial_state->hmi_level(),
-                                  state_to_setup->hmi_level()));
+                OnHMIStateChanged(simple_app_id_, _, state_to_setup));
```

Comment:
@mkorniichuk why did you removed `initial_state` check? I think its necessary to verify as well

Reply:
@AKalinich-Luxoft fixed in 4f71e8ac39d897fdc5afcaef8e44c504fbda9ef3